### PR TITLE
Made some language tweaks to some of the configuration documentation

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
Hey @mfussenegger I think I got the various line lengths and line breaks correct this time, but let me know if I didn't.

I didn't correct all the text here, but was reading a few sections for a blog post and corrected those relevant areas.

### CODE REVIEW

File: docs/configuration.txt [hunk 1]
```diff
@@ -7,36 +7,35 @@ Configuration
...
     sh$ ./bin/crate -Des.config=/path/to/config.yml
 
-Any option can be configured either by the config file or as system
+Any option can be configured either by the config file or as a system
 property. If using system properties the required prefix 'es.' will be
 ignored.
 
-For example, configuring the cluster name by using system properties
-will work this way::
+For example, configuring the cluster name by using system properties works liek
```
Comment (mfussenegger) [inline docs/configuration.txt]: typo in like

**Comment by ChrisChinchilla** [comment]
@mfussenegger corrected.

**Comment by mfussenegger** [comment]
ack, but please squash the 2 commits

**Comment by ChrisChinchilla** [comment]
squashed @mfussenegger

**Comment by ChrisChinchilla** [comment]
And random line breaks removed @mfussenegger

**Comment by mfussenegger** [comment]
sorry forgot about this.

ack

**Comment by ChrisChinchilla** [comment]
Aggh, @mfussenegger again one of the files has been changed multiple times since I committed… This may take me some time to merge or just start again (again)…

**Comment by ChrisChinchilla** [comment]
As there is new content, I think I will just start again on this file.

**Pull request merged**